dask/kubernetes/readme.md

```diff
@@ -0,0 +1,72 @@
+# Exploring cuML algorithms with dask-kubernetes in Google Cloud Platform (GCP)
+
+This guide aims to showcase a joint working example of [`cuML`](https://docs.rapids.ai/api/cuml/stable/),
+[`dask-kubernetes`](https://kubernetes.dask.org/en/latest/index.html) and [Google Cloud Platform](https://cloud.google.com/).
+
+## Prerequisite
+
+Install the following package in your environment:
+- [kubectl] (https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
+- [gcloud] (https://cloud.google.com/sdk/docs/install)
+
+Login to your gcloud account with
+
+```bash
+gcloud init
+```
+
+## Cluster Setup
+
+First,
+let's setup the cluster to supply the resources required by the examples.
+In total,
+we will use at most 1 dask-scheduler pod and 8 dask-cuda-workers.
+See `spec/sched-spec.yaml` and `spec/worker-spec.yaml` for resource requirements.
+
+```bash
+gcloud container clusters create rapids-dask-kubernetes \
+  --accelerator type=a100,count=8 \
+  --zone us-west1-b \
+  --machine-type a2-highgpu-1g
```

Comment:
This instance type has `-1g` on the end but the accelerators say `count=8`. Either the type or count needs updating to match.